VictoriaBankFinalBeta.rb

```diff
@@ -1,96 +1,102 @@
+(0...accounts_arr.size).each do |i|
+  iban = accounts_arr[i][0]
+  transaction_arr = []
   browser.div(class: 'chosen-container chosen-container-single contract-select chosen-container-single-nosearch').click
-  browser.span(text: @iban).click
+  browser.span(text: iban).click
 
-# Wait for page charging 
+  # Wait for page charging.
   sleep(10)
-
-# Fetch and parse HTML document
+  # Initializing variables.
+  day_, date, description, amount, currency = ''
+  # Fetch and parse HTML document
   doc = Nokogiri::HTML(browser.html)
-  doc.xpath('//div[@class="operations"]').each do |div|
+  doc.css('//div[@class="operations"]').each do |div|
     div.search('div[@class="month-delimiter"]').each do |month_div|
       month_year = month_div.text.strip
       div.search('div[@class="day-operations"]').each do |span|
         span.search('div[@class="day-header"]').each do |day|
-          @day_ = day.text.strip.gsub(/[^\d]/, '')
+          day_ = day.text.strip.gsub(/[^\d]/, '')
         end
         span.search('ul[@class="operations-list"]/li').each do |list|
           list.search('span').each do |cell|
-            time = cell.text.strip if cell.attr('class') == ('history-item-time')
-            @date = "#{@day_} " + "#{month_year}" + "#{time}"
-            @discription = cell.text.strip if cell.attr('class') == ('history-item-description')
-            next unless cell.attr('class') == ('history-item-amount transaction ') ||
-                cell.attr('class') == ('history-item-amount transaction income')
+            time = cell.text.strip if cell.attr('class') == 'history-item-time'
+            date = "#{day_} " + "#{month_year}" + "#{time}"
+            description = cell.text.strip if cell.attr('class') == 'history-item-description'
+            next unless cell.attr('class') == 'history-item-amount transaction ' ||
+                cell.attr('class') == 'history-item-amount transaction income'
+
             money = cell.text.strip
-            @amount = money.to_s.scan(/\+?\d+\.\d+/)
-            @currency = money.to_s.scan(/[A-Z]+/)
-            transactionArr.push(Transactions.new(@date, @discription, @amount, @currency, @iban))
+            amount = money.to_s.scan(/\+?\d+\.\d+/)
+            currency = money.to_s.scan(/[A-Z]+/)
+            transaction_arr.push(Transactions.new(date, description, amount, currency, iban))
           end
         end
       end
     end
   end
-  fullAccountArray.push(Accounts.new((acountsArr[i][0]).to_s, (acountsArr[i][1]).to_s,
-                                     (acountsArr[i][2]).to_s, (acountsArr[i][3]).to_s,
-                                     transactionArr))
+  full_account_array.push(Accounts.new((accounts_arr[i][0]).to_s,
+                                       (accounts_arr[i][1]).to_s,
+                                       (accounts_arr[i][2]).to_s,
+                                       (accounts_arr[i][3]).to_s,
+                                       transaction_arr))
 end
```

Comment:
You found a good solution for transaction `Transactions.new(date, description, amount, currency, iban)`  
Make the same object creation here.  
Its hard to see what is `accounts_arr[i][2]`